File: Data Structures & Algorithms/two-integer-sum-ii/submission-13.cs
public class Solution {
    public int[] TwoSum(int[] numbers, int target) {
        int left = 0; int right = numbers.Length;

        for(int i = 0; i < numbers.Length; i++) {
            int sum = numbers[left] + numbers[right];

            if (sum == target) {
                return new int[] {left + 1, right + 1};
            } else if (sum > target) {
                right--;
            } else {
                left++;
            }

        }

        return new int[0];
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-13.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        while(i < len(nums)):
            j = j+1 
            for j in len(nums):
                proposed = num[i] + num[j]
                if proposed == target:
                    return [i, j]
                
        return false